client/extensions/woocommerce/state/sites/settings/stripeConnectAccount/handlers.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import request from 'woocommerce/state/sites/http-request';
+import {
+	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_ERROR,
+	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_REQUEST,
+	WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_SUCCESS,
+} from 'woocommerce/state/action-types';
+
+export default {
+	[ WOOCOMMERCE_SETTINGS_STRIPE_CONNECT_ACCOUNT_CREATE_REQUEST ]: [ dispatchRequest(
+		handleCreateRequest,
+		handleCreateRequestSuccess,
+		handleCreateRequestError
+	) ],
+};
+
+export function handleCreateRequest( { dispatch }, action ) {
+	const { email, countryCode, siteId } = action;
+	dispatch(
+		request( siteId, action, '/wc/v1' ).
+		post( 'connect/stripe/account/', { email, country: countryCode } )
+	);
+}
+
+export function handleCreateRequestSuccess( store, action, { data } ) {
+	const { email, siteId } = action;
+	const { body, status } = data;
+
+	// TODO - is this needed, or will error get fired directly for us?
+	if ( status !== 200 ) {
```

Comment:
This bit is only needed if you are making a request with `&_envelope` (or the soon to come [`getWithHeaders`] (https://github.com/Automattic/wp-calypso/pull/17923). `&_envelop` makes it so the returned HTTP status is always 200, and the real status is returned in `data`. This is needed for things like reviews or orders, since we need pagination data from the headers.

You can drop this since you are not using it, and the data-layer will call `handleCreateRequestError` on those requests.

Reply:
https://developer.wordpress.org/rest-api/using-the-rest-api/global-parameters/#_envelope

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17922/commits/427f3944f4b8a58d59ac7785e0695175bad9aff0